content/applications/marketing/surveys/dashboard.rst

```diff
@@ -0,0 +1,178 @@
+- Percentage and data bar showcasing how many people have :guilabel:`Passed` or become
+  :guilabel:`Certified`
+
+.. note::
+   A half-trophy background image behind the survey name indicates that the survey is a
+   *Certification*.
+
+.. note::
+   The :guilabel:`Passed` percentage and bar **only** appears when a *Required Score* has been
+   configured for that particular survey.
+
+   The :guilabel:`Certfied` percentage and bar **only** appears if that particular survey has the
+   *Is a Certification* option enabled on the survey form.
+
+   If neither :guilabel:`Passed` nor :guilabel:`Certified` appear on the line, that indicates the
+   survey is without a *Required Score* and was not enabled with the *Is a Certification* option.
+
+- Number of :guilabel:`Courses` related to that survey, which **only** appears if more than one
+  course has been created and attached to a single survey
+
+To the far-right of those data points on the survey lines, located on the *Surveys* application
+dashboard, are a collection of buttons.
+
+Those buttons are as follows:
+
+- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` pop-up form that can be used to
+  invite potential participants to the survey - complete with a :guilabel:`Survey Link` that can be
+  copied and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
+
+   .. image:: dashboard/share-survey-popup.png
+      :align: center
+      :alt: The 'Share a Survey' pop-up window that appears in the Odoo Surveys application.
+
+  When the :guilabel:`Send by Email` toggle is 'on' (green switch), additional fields appear, in
+  which :guilabel:`Recipients` and a :guilabel:`Subject` can be added to the email. Below that, a
+  dynamic email template, complete with a :guilabel:`Start Certification` button appears, which can
+  be modified, as well, if needed.
+
+  Once modifications are complete, click :guilabel:`Send` to send that email invite to all the email
+  addresses/contacts listed in the :guilabel:`Recipients` field.
+
+.. image:: dashboard/share-survey-popup-email-toggle.png
+   :align: center
+   :alt: The 'Share a Survey' pop-up window in Odoo Surveys with the Send by Email toggled on.
+
+.. important::
+   Clicking the :guilabel:`Share` button results in an error if the survey line has zero questions.
+
+- :guilabel:`Test`: click to take a test version of the survey in a new tab, from the point-of-view
+  of a survey participant, in order to check for any errors or inconsistences.
+
+- :guilabel:`See results`: click to reveal a new tab showcasing detailed metrics and graphical
+  representations of all survey participants, questions, and responses for deeper analysis.
+
+- :guilabel:`Start Live Session`: click to initiate a *Live Session* survey, and reveal the *Session
+  Manager* window in a new tab. This button is **not** present for surveys that have enabled the
+  *Is a Certification* option on the survey form.
```

Comment:
Remove unnecessary formatting and update to casing to "Live session" as it is on the Odoo UI on Survey backend
```suggestion
- :guilabel:`Start Live Session`: click to initiate a *Live session* survey, and reveal a session
  manager window in a new tab. This button is **not** present for surveys that have enabled the
```